src/wiki/plugins/metadata/migrations/0047_auto_20200808_2256.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.13 on 2020-08-09 02:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
There are 2 new migrations for 1 new field. Could you please delete both these files and rerun makemigrations?

Reply:
Sure